src/test/java/org/broadinstitute/hellbender/utils/GenotypeUtilsUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.variant.variantcontext.*;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class GenotypeUtilsUnitTest {
```

Comment:
good catch, I wonder if I can alter my autogenerated test class template to include that...

looks like InbreedingCoeffUnitTest wasn't extending BaseTest either...